examples/builder.js

```diff
@@ -0,0 +1,21 @@
+const gi = require('../lib/index');
+gi.startLoop()
+
+const Gdk          = gi.require('Gdk', '3.0');
+const Gtk          = gi.require('Gtk', '3.0');
+
+Gtk.init()
+
+const builder = Gtk.Builder.newFromFile('./builderExample.glade');
```

Comment:
`path.join(__dirname, 'builderExample.glade')`, otherwise it doesn't find the file when called from the project root.